alertmanager/alertmanager.go

```diff
@@ -2,46 +2,51 @@ package alertmanager
 
 import (
 	"fmt"
-	"github.com/go-kit/kit/log/level"
 	"net/http"
 	"net/url"
 	"strings"
 	"time"
 
 	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 )
 
+// APIClient of alertmanager
 type APIClient struct {
-	Url            *url.URL
+	URL            *url.URL
 	ConfigPath     string
 	ConfigTemplate string
 	HTTPClient     *http.Client
-	Id             int
+	ID             int
 	Key            string
-	logger     log.Logger
+	logger         log.Logger
 }
 
-type AlertmanagerConfig struct {
+// Config of alertmanager
+type Config struct {
 	Receivers    string
 	Routes       string
 	InhibitRules string
 }
 
-// reload alertmanager
-func (c *APIClient) Reload() (error,int) {
-	return c.doPost(c.Url.String())
+// Reload alertmanager
+func (c *APIClient) Reload() (int, error) {
+	return c.doPost(c.URL.String())
 }
 
 // do post request
-func (c *APIClient) doPost(url string) (error,int) {
+func (c *APIClient) doPost(url string) (int, error) {
 	req, err := http.NewRequest("POST", url, nil)
 	if err != nil {
-		return err, 0
+		return 0, err
 	}
 	resp, err := c.HTTPClient.Do(req)
 	if err != nil {
 		for strings.Contains(err.Error(), "connection refused") {
-			level.Error(c.logger).Log("msg", "Failed to reload alertmanager.yml. Perhaps Alertmanager is not ready. Waiting for 8 seconds and retry again...", "err", err.Error())
+			//nolint:errcheck,lll
```

Comment:
Why do we need `lll`?

Reply:
`lll` is a linter for line length. We disabled it because the error message is intended to be this long.